app/assets/javascripts/tinymce/plugins/alchemy_link/plugin.min.js

```diff
@@ -11,9 +11,9 @@ tinymce.PluginManager.add('alchemy_link', function(editor, url) {
         selection: editor.selection,
         editor: editor
       };
-      var link_dialog = new Alchemy.LinkDialog(link_object);
+      window.dialog = new Alchemy.LinkDialog(link_object);
```

Comment:
Please don't attach objects in to the global JS DOM. 

Alchemy already stores the currently opened dialogs in `Alchemy.currentDialogs`. 
And you can access the current (last) dialog with `Alchemy.currentDialog()`

Could you please try to use this or at least store this on `Alchemy.currentLinkDialog`? Thanks